html-aam/index.html

```diff
@@ -2309,24 +2302,18 @@ <h4 id="el-figure">`figure`</h4>
               </th>
               <td>
                 <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
-                <div class="relations"><span class="type">Relations:</span> `IA2_RELATION_LABELLED_BY` with child <a href="#el-figcaption">`figcaption`</a> element</div>
               </td>
             </tr>
             <tr>
               <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
               <td>
                 <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
-                <div class="general">
-                  Accessible name derived from `figcaption` according to the <a href="#figure-element-accessible-name-computation">`figure` Element Accessible Name Computation</a>
-                </div>
```

Comment:
Sorry I missed this... 

> removes figcaption from the name calculation steps in that section of the spec, and explicitly states in that section that figcaption does not contribute to name or description unless specified by the author (e.g., via aria)

What's the reason for this removal? WebKit and Safari implementations agree on using figcaption as the name if none other is provided via ARIA. Seems logical to me to keep this.

Reply:
here is one of the examples that was referenced as rational for this change [specifically the link to the wicg thread where people are using figcaption to provide transcripts for a video within a figure](https://github.com/w3c/html-aam/pull/359#issuecomment-1017555985)

Reply:
yeah, that's a good example

Reply:
Scott and I chatted, and are working on a updated that will account for [this HTML text](https://html.spec.whatwg.org/multipage/embedded-content.html#the-img-element):

> 2. If the image is a descendant of a figure element that has a child figcaption element, and, ignoring the figcaption element and its descendants, the figure element has no [flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2) descendants other than [inter-element whitespace](https://html.spec.whatwg.org/multipage/dom.html#inter-element-whitespace) and the img element, then return the contents of the first such figcaption element.